### ISSUE TITLE
Footer navigation links are missing a visible focus ring for keyboard navigation

### ISSUE THREAD

**Issue by attaboy** [CivicTechTO/civic-spark.ca#1]
I think this is because the focus ring colour is the same as the background colour. We might want to just roll with the system default which I think avoids this problem!
